locallens_api/locallens_api/products/api/views.py

```diff
@@ -0,0 +1,17 @@
+# shop/views.py
+
+from rest_framework import generics
+
+from locallens_api.products.models import Product
+
+from .serializers import ProductSerializer
+
+
+class ProductListCreateAPIView(generics.ListCreateAPIView):
+    queryset = Product.objects.all()
+    serializer_class = ProductSerializer
+
+
+class ProductRetrieveUpdateDestroyAPIView(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Product.objects.all()
+    serializer_class = ProductSerializer
```

Comment:
if those view are working correctly we can use them instead of the views in products/views.py